examples/CInterface/nonlings_fed2.c

```diff
@@ -1,10 +1,6 @@
 /*
-Copyright (C) 2017-2018, Battelle Memorial Institute
-All rights reserved.
-
-This software was co-developed by Pacific Northwest National Laboratory, operated by the Battelle Memorial
-Institute; the National Renewable Energy Laboratory, operated by the Alliance for Sustainable Energy, LLC; and the
-Lawrence Livermore National Laboratory, operated by Lawrence Livermore National Security, LLC.
+Copyright © 2017-2018, Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC
```

Comment:
This line is longer than would be allowed by clang format so will get changed when I next run that over the files

is it possible to divide this into 2 lines in a nice fashion
maybe
```
Copyright © 2017-2018, 
Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC
```

or 

```
Copyright © 2017-2018, Battelle Memorial Institute; Lawrence Livermore National Security, LLC; 
Alliance for Sustainable Energy, LLC
```

Reply:
I'll divide these lines in the first manner.